src/core/adapters/youtrack.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * YouTrack adapter
+ *
+ * The adapter uses the DOM to extract ticket information from issues.
+ */
+
+import type { TicketData } from "../types";
+import { $has, $text } from "./dom-helpers";
+import { hasRequiredDetails } from "./utils";
+
+async function scan(url: URL, document: Document): Promise<TicketData[]> {
+  if (!$has(".yt-issue-view", document)) return [];
+
+  const id = $text(".js-issue-id", document)?.trim();
+  const title = $text("[data-test='issueSummary']", document);
+  const type =
+    $text("[data-test='Type']", document)?.toLowerCase().trim() || "feature";
```

Comment:
Weird that the word "type" is capitalized here. But you'll probably notice quite fast when then changes...